module/zfs/arc.c

```diff
@@ -2478,7 +2504,7 @@ arc_loan_inuse_buf(arc_buf_t *buf, void *tag)
 	(void) refcount_add(&hdr->b_l1hdr.b_refcnt, arc_onloan_tag);
 	(void) refcount_remove(&hdr->b_l1hdr.b_refcnt, tag);
 
-	atomic_add_64(&arc_loaned_bytes, -arc_buf_size(buf));
+	arc_loaned_bytes_update(arc_buf_size(buf));
```

Comment:
Good observation.  Yes, the existing code was wrong here and we need to drop the minus sign.  The `arc_loan_buf*` and `arc_return_buf` functions should increment and decrement this value accordingly.